update/Dockerfile

```diff
@@ -40,18 +40,31 @@ RUN --mount=type=secret,id=STEAM_USERNAME \
         i=$(( i+1 )); \
     done; \
     echo "[UPDATE] Performing cleanup"; \
-    cd "$STEAMCMD_DIR" && rm -rf \
-        linux64 \
-        package \
-        public \
-        siteserverui \
-        steam \
-        linux32/libtier0_s.so \
-        linux32/libvstdlib_s.so \
-        linux32/steamclient.so \
-        linux32/steamconsole.so \
-        update_hosts_cached.vdf; \
-    rm -rf /root/.steam; \
+    # Clean up a different set of files for srcds/cs2
+    if  [ "$APPID" = 730 ]; then \
+        cd "$STEAMCMD_DIR" && rm -rf \
+            package \
+            public \
+            siteserverui \
+            steam \
+            linux32/libtier0_s.so \
+            linux32/libvstdlib_s.so \
+            linux32/steamconsole.so \
+            update_hosts_cached.vdf; \
```

Comment:
it needs to remove everything in `/root/.steam/` except the symlinks to binaries. The folder may contain credentials. Alternatively, use `+logout` ?

Reply:
I did try building a clean and updating an existing image. The directory isn't created in either scenarios. Removed the line from `update/Dockerfile` especially because `update` removed the necessary symlinks from `build`

Reply:
Either way the symlinks just need to be created on `build` like presently and the game should start fine

Reply:
might be named `config/config.vdf`, may be in the game folder, or `~/.steam` or other places, not sure. pretty sure its cached somewhere?

Reply:
let me try to find out

Reply:
it appears, cleaning `~/Steam` cleans out the user data.

Reply:
added comments.